internal/unionstore/pipelined_memdb.go

```diff
@@ -375,17 +380,17 @@ func (p *PipelinedMemDB) SnapshotGetter() Getter {
 
 // Staging is not supported for PipelinedMemDB, it returns 0 handle.
```

Comment:
The comments should be updated, better to include the correctness explanation as well.